### PR TITLE
net-im/yowsup: New package

### PR DESCRIPTION
**Pull request by ConiKost**
net-im/yowsup2: New package

Depends on: https://bugs.gentoo.org/608882
Closes: https://bugs.gentoo.org/652258

### CODE REVIEW

File: net-im/yowsup2/yowsup2-2.5.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+src_prepare() {
+	# Remove argparse as dependency, as it's built in
+	sed -i -e "s/, 'argparse'//" setup.py || die
+
+	# Allow newer version of dev-python/six
+	sed -i -e 's/six==1.10/six>=1.10/' setup.py || die
+
+	# Apply user patches
```
Comment (Zlogene) [inline net-im/yowsup2/yowsup2-2.5.7.ebuild]: useless

Comment (ConiKost) [reply net-im/yowsup2/yowsup2-2.5.7.ebuild]: Fixed.

File: net-im/yowsup2/yowsup2-2.5.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/axolotl-curve25519[${PYTHON_USEDEP}]
+	dev-python/configargparse[${PYTHON_USEDEP}]
+	dev-python/pillow[${PYTHON_USEDEP}]
+	dev-python/protobuf-python[${PYTHON_USEDEP}]
+	dev-python/pycrypto[${PYTHON_USEDEP}]
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	dev-python/setuptools"
```
Comment (mgorny) [inline net-im/yowsup2/yowsup2-2.5.7.ebuild]: PYTHON_USEDEP?

File: net-im/yowsup2/yowsup2-2.5.7.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="${PN: :-1}"
```
Comment (mgorny) [inline net-im/yowsup2/yowsup2-2.5.7.ebuild]: Then why are you naming it `yowsup2`?

Comment (ConiKost) [reply net-im/yowsup2/yowsup2-2.5.7.ebuild]: No clue, renamed it, makes more sense.

File: net-im/yowsup2/yowsup2-2.5.7.ebuild (hunk 2 above)
Comment (ConiKost) [reply net-im/yowsup2/yowsup2-2.5.7.ebuild]: Thanks, changed.

**Comment by ConiKost** [comment]
Rebased for green CI, since all deps are merged into portage.

**Pull request closed without merging**